### PR TITLE
add mcsucht.de

### PR DESCRIPTION
**Pull request by Nik2810-Original**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by Nik2810-Original** [comment]
[manifest.json](https://github.com/LabyMod/server-media/files/14894954/manifest.json)

**Comment by Nik2810-Original** [comment]
[manifest.json](https://github.com/LabyMod/server-media/files/14894956/manifest.json)

**Comment by Nik2810-Original** [comment]
[manifest.json](https://github.com/LabyMod/server-media/files/14894959/manifest.json)

**Comment by Nik2810-Original** [comment]
https://github.com/Nik2810-Original/server-media/blob/master/minecraft_servers/mcsucht/manifest.json

**Comment by Nik2810-Original** [comment]
minecraft_servers/mcsucht/manifest.json

**Comment by Nik2810-Original** [comment]
minecraft_servers/mcsucht/manifest.json

File: minecraft_servers/mcsucht/manifest.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "mcsucht",
+  "nice_name": "MCSUCHT",
+  "direct_ip": "mcsucht.de",
+  "server_wildcards": [
+    "%.mcsucht.de"
```
Comment (maaaathis) [inline minecraft_servers/mcsucht/manifest.json:6]: Doesn't look like a real wildcard
image

File: minecraft_servers/mcsucht/gamemodes/citybuild/icon.png
Comment (maaaathis) [inline minecraft_servers/mcsucht/gamemodes/citybuild/icon.png:1]: You cannot add custom images for gamemodes. Remove the hole `gamemodes/` folder.

File: minecraft_servers/mcsucht/gamemodes/citybuildcreative/icon.png
Comment (maaaathis) [inline minecraft_servers/mcsucht/gamemodes/citybuildcreative/icon.png:1]: You cannot add custom images for gamemodes. Remove the hole `gamemodes/` folder.

File: minecraft_servers/mcsucht/gamemodes/lifestealsmp/icon.png
Comment (maaaathis) [inline minecraft_servers/mcsucht/gamemodes/lifestealsmp/icon.png:1]: You cannot add custom images for gamemodes. Remove the hole `gamemodes/` folder.

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/mcsucht/gamemodes/citybuild/icon.png
Comment (DoJapHD) [reply minecraft_servers/mcsucht/gamemodes/citybuild/icon.png:1]: It can https://i.imgur.com/gMteOiG.png

Comment (maaaathis) [reply minecraft_servers/mcsucht/gamemodes/citybuild/icon.png:1]: Sorry, my bad

Comment (Nik2810-Original) [reply minecraft_servers/mcsucht/gamemodes/citybuild/icon.png:1]: Do I have to change something?

**Comment by Nik2810-Original** [comment]
minecraft_servers/mcsucht/manifest.json

File: minecraft_servers/mcsucht/manifest.json (hunk 1 above)
Comment (Nik2810-Original) [reply minecraft_servers/mcsucht/manifest.json]: Sorry, I removed the wildcard

**Comment by DoJapHD** [comment]
Server is in maintenance mode; Reopen PR once server is back online

**Comment by Nik2810-Original** [comment]
It could take some time for the server to come out of maintenance mode (1-2 months). Is this a problem?

**Pull request closed without merging**